karrot/activities/tests/test_renderers.py

```diff
@@ -0,0 +1,31 @@
+from django.test import TestCase
+from rest_framework.utils.serializer_helpers import ReturnDict, ReturnList
+from rest_framework.response import Response
+from karrot.activities.renderers import ICSCalendarRenderer
+
+
+class ICSCalendarRendererTest(TestCase):
+    def setUp(self):
+        self.renderer = ICSCalendarRenderer()
+        self.event = ReturnDict({
+            'DTSTART': '20210319T170000Z',
+            'DTEND': '20210319T180000Z',
+            'DESCRIPTION': 'Hello\nWorld!',
+            'TRANSP': 'OPAQUE'
+        },
+                                serializer=None)
+        self.expected_output = (
+            "BEGIN:VCALENDAR\r\n" + "VERSION:2.0\r\n" + "PRODID:-//Karrot//EN\r\n" + "BEGIN:VEVENT\r\n" +
+            "DTSTART:20210319T170000Z\r\n" + "DTEND:20210319T180000Z\r\n" + "DESCRIPTION:Hello\\nWorld!\r\n" +
+            "TRANSP:OPAQUE\r\n" + "END:VEVENT\r\n" + "END:VCALENDAR"
```

Comment:
I guess yapf formatted this a bit oddly! Maybe a list of values joined by `\r\n` would be easier to read?